test/aria/widgets/form/autocomplete/popupWidth/AdaptToContentWidthTest.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2013 Amadeus s.a.s.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Aria.classDefinition({
+    $classpath : "test.aria.widgets.form.autocomplete.popupWidth.AdaptToContentWidthTest",
+    $extends : "aria.jsunit.RobotTestCase",
+    $dependencies : ["aria.popups.PopupManager", "aria.utils.Dom"],
+    $constructor : function () {
+        this.$RobotTestCase.constructor.call(this);
+        this.setTestEnv({
+            template : "test.aria.widgets.form.autocomplete.popupWidth.AdaptToContentWidthTpl"
+        });
+
+    },
+    $prototype : {
+
+        runTemplateTest : function () {
+            var field = this.getInputField("ac");
+            this.synEvent.execute([["click", field], ["type", field, "a"]], {
+                fn : this._afterTyping,
+                scope : this
+            });
+        },
+
+        _afterTyping : function () {
+            var testCase = this;
+
+            this.waitFor({
+                condition : function () {
+                    return !!testCase.getWidgetDropDownPopup("ac");
+                },
+                callback : function () {
+                    testCase.$callback({
+                        fn : testCase._afterDropdownOpen,
+                        scope : testCase
+                    });
+                }
+            });
+        },
+
+        _afterDropdownOpen : function () {
+            this._checkDropdownWidth();
+            var field = this.getInputField("ac");
+            this.synEvent.execute([["type", field, "[back]a"]], {
```

Comment:
The `[back]` key is supported in the `aria.jsunit.RobotPhantomJS` implementation of the Robot but not in the implementations extending `aria.jsunit.RobotJava`. Maybe you should use `[BACK_SPACE]` instead.